### DISCUSSION TITLE
Feature request: replace Z fine adjustment 'undo' button with a button that issues Z_OFFSET_APPLY_PROBE

### DISCUSSION THREAD

**Discussion by axg20202** [KlipperScreen/KlipperScreen#1524] [Ideas]
Title says it all really. The current 'undo' button for fine tuning of z offset is completely redundant, but there is currently no button to write on-the-fly z tuning to config. This feature would still require a save_config after the print but that is normal behaviour. Being able to permanently apply fine z adjustment in klipperscreen is a much needed feature.

**Comment by alfrix** [comment]
i do use that z reset button sometimes

if it requires a manual save_config after the print you can use the current provided buttons in the print end.

if the print status was closed, the save config button can be modified to ask if you want to apply the offset

as an alternative i thought about just adding the 'Job Status' to the 'More' menu even when not printing, that would let you access this buttons, same idea for 'Fine Tune'

**Comment by axg20202** [comment]
Makes sense. I was aware of the save config button. think it could be a good workaround to have this button prompt for an optional save of the new z offset, but I still think it would be preferable to be able to apply the offset at the time the changes are being made. Perhaps a 'do you want to apply z offset and save config?' prompt appearing at print end could be an option, but I appreciate this might be challenging to implement. The reason I suggest this is it's easy to forget to apply and save z fine tunes at print end.

On the subject of the More button, have you considered renaming Move to Motion? I find I often push More when I actually want Move because at a glance they look similar.
